backend/baseclass.pm

```diff
@@ -1017,13 +999,6 @@ sub _time_to_assert_screen_deadline {
     return $self->assert_screen_deadline - time;
 }
 
-sub reduce_deadline {
```

Comment:
Let's assume no test distributions use this (the openSUSE one doesn't).

Reply:
Well, backend/baseclass isn't exactly an "outwards facing API" so I don't have doubts removing this :)